src/components/10-atoms/toggle-switch/react/demo.jsx

```diff
@@ -7,6 +7,7 @@ import changelog from '../CHANGELOG.md';
 import readme from '../README.md';
 import AXAToggleSwitchReact from './AXAToggleSwitchReact';
 import ControlledWithOnChange from './ControlledWithOnChange';
+import ControlledWithDummyOnChange from './ControlledWithDummyOnChange';
```

Comment:
Dummy doesn't quite describe the constant active value. No strong objection, but the point here is the constant active value, not the onChange, is it not?